src/util.hpp

```diff
@@ -115,10 +105,9 @@ std::pair<double, double> convert_vt_to_ll(std::uint32_t extent,
   The first point is considered the "origin" and its latitude is used to initialize
   the ruler. The second is considered the "feature" and is the distance to.
 */
-namespace cr = mapbox::cheap_ruler;
-double distance_in_meters(cr::point origin_lnglat, cr::point feature_lnglat) {
+double distance_in_meters(mapbox::geometry::point<double> origin_lnglat, mapbox::geometry::point<double> feature_lnglat) {
```

Comment:
The input here should likely be passed by `const &`

Reply:
Done!